src/api/ws/websocket-api.ts

```diff
@@ -0,0 +1,191 @@
+import WebSocket from 'ws'
+import {
+    AnyFunction,
+    AnyFunctionWithThis,
+    OnGetAllowedMethodsCb,
+    OnMessageCb,
+    OnOrderCb,
+    OnOrderCreatedCb,
+    OnOrderFilledCb,
+    OnOrderFilledPartiallyCb,
+    OnOrderInvalidCb,
+    OnOrderNotEnoughBalanceOrAllowanceCb,
+    OnPongCb,
+    OrderEventType,
+    RpcEventType,
+    WsApiConfig
+} from './types'
+
+export const orderEvents: OrderEventType['event'][] = [
+    'order_created',
+    'order_invalid',
+    'order_balance_or_allowance_change',
+    'order_filled',
+    'order_filled_partially'
+]
+
+export class WebSocketApi {
```

Comment:
1. Rename `ws` folder to `ws-api`
2. Move to a higher level
3. Make `ActiveOrdersWebSocketApi` class:
- `WebSocketClient` class and `IWebSocketClient` interface which contains of basic methods like `on`, `send`, 'onError', 'onMessage' etc... in `connector` folder (example: http-provider.connector)
- `ActiveOrdersWebSocketApi` with `onOrder`, 'onOrderCreated' etc.. methods in `ws-api` folder